build.gradle

```diff
@@ -251,6 +251,16 @@ dependencies {
     compile group: 'org.springframework.boot', name: 'spring-boot-starter-web', version: versions.springBoot
     compile group: 'org.springframework.cloud', name: 'spring-cloud-starter-netflix-hystrix', version: versions.springHystrix
 
+    compile('org.apache.logging.log4j:log4j-to-slf4j:2.13.2') {
```

Comment:
@raghera do we want to force log4j or keep suppression? I think suppressing is fine because it is related to SMTP appender which we do not use. What do you think?

Reply:
@makowalski As per the discussion I have removed the forced dependency for the above